src/app/view/components/dot-add-persona-dialog/dot-create-persona-form/dot-create-persona-form.component.html

```diff
@@ -0,0 +1,37 @@
+<form [formGroup]="form" >
+    <div class="form__group">
+        <label class="form__label">{{messagesKey['modes.persona.host']}}: </label>
+        <dot-site-selector (change)="siteChange($event)"></dot-site-selector>
```

Comment:
There is a component for site selector that already have the control value accessor: `dot-site-selector-field` is the same we use in edit content type form.